apolline-flutter/lib/twins/SensorTwin.dart

```diff
@@ -115,54 +112,52 @@ class SensorTwin {
     _callbacks[event] = callback;
   }
 
+  /// Redistributes sensor status updates to registered callbacks.
+  Future<void> _setUpStatusListener () async {
+    this._deviceStream = FlutterReactiveBle().connectToDevice(id: _device.id).listen((ConnectionStateUpdate status) {
+      this._currentState = status.connectionState;
```

Comment:
In case of disconnection, application should regularly try to reconnect to sensor.